build/deploy/mkimage.sh

```diff
@@ -13,19 +13,16 @@ set -ex
 cd -P "$(dirname $0)"
 DIR=$(pwd -P)
 
-rm -rf resources ui cockroach
-mkdir -p build
+rm -rf build && mkdir -p build
 docker run -v "$(pwd)/build":/build "cockroachdb/cockroach-dev" shell "cd /cockroach && \
```

Comment:
Is there a reason for using `$(pwd)` here instead of `${DIR}`? Same question applies elsewhere in this file.